Rockerfile

```diff
@@ -3,7 +3,7 @@
 {{ $tag:= (or .Env.DOCKER_TAG "local") }}
 
 FROM golang:1.7-alpine
-RUN apk --update add bash
+RUN apk --update add bash git
```

Comment:
I guess we don't actually need this step. I could revert it if you prefer. That is only needed to fetch the test dependency. So i could move it to the travis.sh.